bundles/org.openhab.binding.evcc/README.md

```diff
@@ -0,0 +1,171 @@
+# evcc Binding
+
+This binding integrates [evcc - electric vehicle charging control](https://evcc.io), a project that provides a control center for electric vehicle charging.
+
+evcc controls your wallbox(es) with multiple charging modes and allows you to charge your ev with your photovoltaik's excess current.
+To provide an intelligent charging control, evcc supports over 30 wallboxes and over 20 energy meters/home energy management systems from many manufacturers as well as electric vehicles from over 20 car manufacturers.
+Furthermore, evcc calculates your money savings.
+
+This binding enables openHAB to retrieve status data from your evcc installation and to control the charging process.
+For more advanced features like calculated savings, you have to visit the web UI of evcc.
+
+## Supported Things
+
+- `device`: A running evcc installation.
+
+## Discovery
+
+No auto discovery supported.
+
+## Thing Configuration
+
+### `device` Thing Configuration
+
+| Parameter       | Type   | Description                                              | Advanced | Required |
+|-----------------|--------|----------------------------------------------------------|----------|----------|
+| url             | String | URL of evcc web UI, e.g. *https://demo.evcc.io*          | false    | true     |
+| refreshInterval | Number | Interval the status is polled in seconds (minimum is 15) | true     | false    |
```

Comment:
true/false could be replaced by Yes/No or something else which would be more user-oriented. :)